### PR TITLE
Allow disabling !forceready via get5_allow_force_ready.

### PR DESCRIPTION
**Pull request by nickdnk**
I was looking at the forks people have made of get5 to figure out which low-hanging fruits people felt were missing, and one of them had simply removed access to the `!forceready` command, so I've added this as an option.

I also noticed that the message informing players that they can `!forceready` is misleading with regards to the "if you have less than X players" part, as it's only accessible if you have at least `min_players_to_ready` **and** it is also accessible if your team is full and someone just didn't ready, so I removed that part of the message using my Google-translate skills.

@Saph1s Please check the Russian.
@SidiBecker Please check the Portuguese.
@enerbewow Please check the Hungarian.

You can also add the `ForceReadyDisabled` for your language and I'll put that in. The English version is:

`The force-ready command is disabled, but can enabled with get5_allow_force_ready.`

### CODE REVIEW

File: translations/chi/get5.phrases.txt [hunk 1]
```diff
@@ -122,7 +122,7 @@
     }
     "ForceReadyInfoMessage"
     {
-        "chi"            "如果您有不到{1}名玩家，您可以输入{2}来强制准备您的队伍。"
+        "chi"            "您可以输入{1}来强制准备您的队伍。"
```
Comment (nickdnk) [inline translations/chi/get5.phrases.txt:125]: According to Google, this variable was in the wrong place, so I've best-efforted it to be correct.

**Review by Saph1s — CHANGES_REQUESTED** [review]

**Comment by enerbewow** [comment]
I'll need some time, cause my free time is stricted nowdays, maybe in this week I can have some free time to check it

**Comment by nickdnk** [comment]
> I'll need some time, cause my free time is stricted nowdays, maybe in this week I can have some free time to check it

Fair enough, but all you need to do here is:

1. Answer if this is correct, as I removed the last part of the sentence:

    "**Beírhatod a {1} parancsot, ha fel szeretnéd készíteni a csapatod.**"
    
2. Translate this sentence:

    "**The force-ready command is disabled, but can enabled with get5_allow_force_ready.**"
    
 You can just comment here and I'll put it in the code. No actual work required on your part.

**Comment by enerbewow** [comment]
> > I'll need some time, cause my free time is stricted nowdays, maybe in this week I can have some free time to check it
> 
> Fair enough, but all you need to do here is:
> 
> 1. Answer if this is correct, as I removed the last part of the sentence:
>    "**Beírhatod a {1} parancsot, ha fel szeretnéd készíteni a csapatod.**"
> 2. Translate this sentence:
>    "**The force-ready command is disabled, but can enabled with get5_allow_force_ready.**"
> 
> You can just comment here and I'll put it in the code. No actual work required on your part.

1. **"Beírhatod a {1} parancsot, ha fel szeretnéd készíteni a csapatod."**
Beírhatod - **You can type** , **a** - the , **{command}** , parancsot - **command**, ha - **if**, fel szeretnéd - **you want to**, készíteni - **prepare**, a csapatod - **your team** ---  This is correct✔

ha kevesebb mint {2} játékosod van. --- this means: if you have, less then {2} player.

2. **"The force-ready command is disabled, but can enabled with get5_allow_force_ready."**

HUN Translate:
A !forceready parancs ki van kapcsolva, de engedélyezheted a get5_allow_force_ready paranccsal."
force-ready is not interpret in Hungarian. In raw backtrasnlate is ready with power (by some external force).

**Comment by nickdnk** [comment]
> "Beírhatod a {1} parancsot, ha fel szeretnéd készíteni a csapatod."
Beírhatod - You can type , a - the , {command} , parancsot - command, ha - if, fel szeretnéd - you want to, készíteni - prepare, a csapatod - your team --- This is correct✔

Great. Thanks.

> HUN Translate: A `!forceready` parancs ki van kapcsolva, de engedélyezheted a get5_allow_force_ready paranccsal." force-ready is not interpret in Hungarian. In raw backtrasnlate is ready with power (by some external force).

You are right, we should inject the variable into the sentence (!forceready) instead of trying to translate it ("the force-ready command"). I will make this change to the English version.

**Comment by nickdnk** [comment]
@Saph1s 

`Команда для принудительной подготовки отключена, но ее можно включить с помощью {2}.` must have the `!forceready` command injected as the first parameter instead, i.e.: `The !forceready command is disabled, but can be enabled with get5_allow_force_ready.`

**Review by SidiBecker — CHANGES_REQUESTED** [review]

File: translations/pt/get5.phrases.txt [hunk 2]
```diff
@@ -198,7 +198,7 @@
     }
     "ForceReadyInfoMessage"
     {
-        "pt"            "Você pode digitar {1} para forçar seu time a ficar pronto se você tiver menos de {2} jogador."
+        "pt"            "Você pode digitar {1} para forçar seu time a ficar pronto."
     }
```
[the remark this answers is not in the document]
Comment (SidiBecker) [reply translations/pt/get5.phrases.txt:202]: Sorry for the delay,  I forgot to submit my review days ago 🙄

File: translations/ru/get5.phrases.txt [hunk 3]
```diff
@@ -198,7 +198,11 @@
     }
     "ForceReadyInfoMessage"
     {
-        "ru"            "Вы можете написать {1}, чтобы принудительно подготовить свою команду, если у вас меньше {2} игроков."
+        "ru"            "Вы можете написать {1}, чтобы принудительно подготовить свою команду."
+    }
+    "ForceReadyDisabled"
+    {
+        "ru"            "Команда для принудительной подготовки отключена, но ее можно включить с помощью {2}."
```
Comment (Saph1s) [inline translations/ru/get5.phrases.txt:205]: "Команда {1} отключена, но ее можно включить с помощью {2}."

**Review by Saph1s — CHANGES_REQUESTED** [review]

File: translations/pt/get5.phrases.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickdnk) [reply translations/pt/get5.phrases.txt:202]: Hey @SidiBecker. 

Thanks!

See my latest comment. The "!forceready" command must be injected into the `ForceReadyDisabled`-string as the first argument.

[the remark this answers is not in the document]
Comment (SidiBecker) [reply translations/pt/get5.phrases.txt:202]: In this case:
```
"ForceReadyDisabled"
    {
        "pt"            "O comando {1} está desativado, mas pode ser ativado com {2}."
    }
```

Translation of  **"The {1} command is disabled, but can be enabled with {2}."**

**Review by SidiBecker — CHANGES_REQUESTED** [review]

**Comment by nickdnk** [comment]
I think we're good now with the translators we have.

**Pull request merged**